gradle/versions.gradle

```diff
@@ -0,0 +1,39 @@
+// Variables for entire project
+ext {
+  // Setup
+  minSdkVersion = 14
+  targetSdkVersion = 25
+  compileSdkVersion = 25
+  buildToolsVersion = "25.0.0"
+
+  // Common versions
+  supportLibraryVersion = "25.0.0"
+}
+
+// Plugins
+ext.plugs = [
+    "gradle"                  : "com.android.tools.build:gradle:2.2.3",
+    "androidMavenGradlePlugin": "com.github.dcendents:android-maven-gradle-plugin:1.4.1"
+]
+
+// Dependencies
+ext.deps = [
```

Comment:
Love the version change, it helps us to have a single place for all the versions. Thanks!! 
But the dependency part, it just introduce one extra level, do we really need it?